hoover/site/settings/common.py

```diff
@@ -152,3 +159,9 @@
 CELERY_BROKER_URL = os.getenv('SEARCH_AMQP_URL')
 
 SNOOP_FORWARD_HEADERS = ['Content-Disposition', 'Accept-Ranges', 'Content-Range', 'Content-Length']
+
+TUS_UPLOAD_DIR = os.path.join(base_dir, 'tus_upload')
+TUS_DESTINATION_DIR = os.path.join(base_dir, 'media', 'uploads')
+TUS_FILE_NAME_FORMAT = 'increment'  # Other options are: 'random-suffix', 'random', 'keep'
+TUS_EXISTING_FILE = 'error'  # Other options are: 'overwrite',  'error', 'rename'
+DATA_UPLOAD_MAX_MEMORY_SIZE = 5242880
```

Comment:
Please write as `5  * 2**10  # 5 MB` to avoid mental gymnastics when reading/editing.

Where does this come from? It's a bit bigger than the size of the upload chunk from tus, right? Pls comment